libraries/rib-base/src/main/java/com/badoo/ribs/core/customisation/RibCustomisationDirectoryImpl.kt

```diff
@@ -9,20 +9,14 @@ open class RibCustomisationDirectoryImpl(
 
     private val map: MutableMap<Any, Any> = hashMapOf()
 
-    inline fun <reified T : Any> put(value: T) {
-        put(T::class, value)
+    fun <T : Any> put(value: T) {
+        map[value::class] = value
     }
 
     override fun <T : RibCustomisation> put(key: KClass<T>, value: T) {
         map[key] = value
     }
 
-    fun <T : Any> put(vararg values: T) {
-        values.forEach {
-            map[it::class] = it
-        }
-    }
-
```

Comment:
We even the vararg one in practice, let's keep this one instead of the other!

Reply:
Done